### PR TITLE
Limit checks

### PR DESCRIPTION
**Pull request by MuhammadKhalilzadeh**
`checkController.js` and `MongoDB.js` : Limiting getChecks to only fetch the last 25 checks to display on the bar chart.
`checkController.js` : Error handling
`messages.js` : New message constant, `CHECKS_NOT_FOUND`

### CODE REVIEW

**Review by ajhollid — CHANGES_REQUESTED** [review]
Good starting point!

Error handling should not be done at the Controller level, we have middleware to do that. If you handle the error here we will not have system logs for the error.

The getChecks method needs a bit more flexibility to work as we need it, please see details in comments!

File: Server/controllers/checkController.js [hunk 1]
```diff
@@ -44,9 +44,15 @@ const getChecks = async (req, res, next) => {
...
-    return res
-      .status(200)
-      .json({ success: true, msg: successMessages.CHECK_GET, data: checks });
+    if (checks) {
+      return res
+        .status(200)
+        .json({ success: true, msg: successMessages.CHECK_GET, data: checks });
+    } else {
+      return res
+        .status(404)
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Server/controllers/checkController.js:53]: So I'll remove the error handling part from here then

File: Server/db/MongoDB.js [hunk 2]
```diff
@@ -416,7 +416,7 @@ const createCheck = async (checkData) => {
 
 const getChecks = async (monitorId) => {
   try {
-    const checks = await Check.find({ monitorId });
+    const checks = await Check.find({ monitorId }).sort({ createdAt: -1 }).limit(25);
```
[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Server/db/MongoDB.js:419]: Yep. Option 1 is the best

[the remark this answers is not in the document]
Comment (MuhammadKhalilzadeh) [reply Server/db/MongoDB.js:419]: Where do you want the parameter then? inside `req.body` or `req.params` or any other options?

[the remark this answers is not in the document]
Comment (ajhollid) [reply Server/db/MongoDB.js:419]: Probably `req.query` i would imagine.  URL would look something like

`/api/v1/monitor/{monitor_id}?limit=25`

File: Server/db/MongoDB.js [hunk 3]
```diff
@@ -414,9 +414,13 @@ const createCheck = async (checkData) => {
  * @throws {Error}
  */
 
-const getChecks = async (monitorId) => {
+const getChecks = async (monitorId, limitedTo) => {
```
Comment (MuhammadKhalilzadeh) [inline Server/db/MongoDB.js:417]: @ajhollid Is this okay?

Comment (ajhollid) [reply Server/db/MongoDB.js:417]: Sure, but I think you can name that var `limit`

Comment (MuhammadKhalilzadeh) [reply Server/db/MongoDB.js]: Done.
how about other parts?

Comment (ajhollid) [reply Server/db/MongoDB.js]: Have you updated the route to include the query param?  I don't think I saw that here

Comment (ajhollid) [reply Server/db/MongoDB.js]: Frontend will also need to be updated to make the appropriate request, and docs need to be updated too

**Comment by MuhammadKhalilzadeh** [comment]
@ajhollid  
Modifications are applied as discussed

**Review by ajhollid — APPROVED** [review]

**Pull request merged**